### PR TITLE
graphics-scripting fixes

### PR DESCRIPTION
**Pull request by humbletim**
... staging PR for discussion with @sethalves @ctrlaltdavid

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7602/HighFidelity-Beta-PR12525-63fd12ce8c3a534b81280c63940d6fcc7a77ed3c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7602/HighFidelity-Beta-PR12525-63fd12ce8c3a534b81280c63940d6fcc7a77ed3c.exe

File: libraries/graphics-scripting/src/graphics-scripting/GraphicsScriptingInterface.cpp [hunk 1]
```diff
@@ -247,8 +247,12 @@ scriptable::ScriptableMeshPointer GraphicsScriptingInterface::newMesh(const QVar
     return scriptable::make_scriptowned<scriptable::ScriptableMesh>(mesh, nullptr);
 }
 
-QString GraphicsScriptingInterface::exportModelToOBJ(const scriptable::ScriptableModel& _in) {
-    const auto& in = _in.getConstMeshes();
+QString GraphicsScriptingInterface::exportModelToOBJ(const scriptable::ScriptableModelPointer& _in) {
```
Comment (ZappoMan) [inline libraries/graphics-scripting/src/graphics-scripting/GraphicsScriptingInterface.cpp]: I know this should have been caught in the prior code review, but `_` should not be used for a prefix for an argument variable. Please repair this.

According to the coding standard  `_` is only used as a prefix for private or protected member variables.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7643/app_12525_PR.apk

File: libraries/graphics-scripting/src/graphics-scripting/GraphicsScriptingInterface.cpp (hunk 1 above)
Comment (humbletim) [reply libraries/graphics-scripting/src/graphics-scripting/GraphicsScriptingInterface.cpp]: Good catch -- fixed in latest commit.  Also scanned for and fixed one other case of a lingering _argument name.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7643/HighFidelity-Beta-PR12525-a85d6b8c86f37274393df9b82672caa2a436da4c.dmg

**Comment by AndrewMeadows** [comment]
Jenkins, rebuild this please.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7709/HighFidelity-Beta-PR12525-a85d6b8c86f37274393df9b82672caa2a436da4c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7709/HighFidelity-Beta-PR12525-a85d6b8c86f37274393df9b82672caa2a436da4c.exe

**Comment by humbletim** [comment]
these changes are now available as part of PR #12876

**Pull request closed without merging**